generated/nidaqmx/nidaqmx.proto

```diff
@@ -3672,7 +5570,7 @@ enum ChannelInt32AttributeValues {
   CHANNEL_INT32_LEVEL1_LOW = 10214;
   CHANNEL_INT32_LOGIC_FAMILY_2POINT_5_V = 14620;
   CHANNEL_INT32_LOGIC_FAMILY_3POINT_3_V = 14621;
-  CHANNEL_INT32_LOGIC_FAMILY_5_V = 14619;
+  CHANNEL_INT32_LOGIC_FAMILY_5V = 14619;
```

Comment:
Same as `LOGIC_FAMILY_5V` above.

Reply:
This change is not okay.

Reply:
Added a fix in https://github.com/ni/grpc-device-scrapigen/pull/127/commits/2ff22d5b025bf6287e43aff545216a1ba34eca4a